app/web/pages/unsubscribe.tsx

```diff
@@ -0,0 +1,21 @@
+import { appGetLayout } from "components/AppRoute";
+import Unsubscribe from "features/auth/notifications/Unsubscribe";
+import { GetStaticProps } from "next";
+import nextI18nextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export const getStaticProps: GetStaticProps = async ({ locale }) => ({
```

Comment:
There's a helper function for this in `i18n/server-side-translations.ts`: `translationStaticProps`